src/Language/PureScript/CodeGen/JS/Printer.hs

```diff
@@ -119,6 +120,30 @@ literals = mkPattern' match'
     , mconcat <$> forM com comment
     , prettyPrintJS' js
     ]
+  match (Pure _ js) = mconcat <$> sequence 
+    [ return $ emit  "/* #__PURE__ */ "
+    , prettyPrintJS' js 
+    ]
+  match (Import _ ident from) = return . emit $
+    "import * as " <> ident <> " from " <> prettyPrintStringJS from
+  match (Export _ idents from) = mconcat <$> sequence
+    [ return $ emit "export {\n"
+    , withIndent $ do
+        let exportsStrings = emit . exportedIdentToString from <$> idents
+        indentString <- currentIndent
+        return . intercalate (emit ",\n") . NEL.toList $ (indentString <>) <$> exportsStrings
+    , return $ emit "\n"
+    , currentIndent
+    , return . emit $ "}" <> maybe "" ((" from " <>) . prettyPrintStringJS) from
+    ]
+    where
+    exportedIdentToString Nothing ident
+      | nameIsJsReserved ident || nameIsJsBuiltIn ident
+      = "$$" <> ident <> " as " <> ident
+    exportedIdentToString _ "$main"
+      = T.concatMap identCharToText "$main" <> " as $main"
```

Comment:
I'm not totally sure what this is about? Isn't this an identity?

Reply:
@sigma-andex Any comments on this?

Reply:
Apparently `$main` is generated by the REPL (PSCI volatile module) and needs to be aliased. I just tested and without it the repl doesnt work. As far as I understood the $main is exported as $dollarmain (due to identCharToText) and because of this needs to be aliased.

Reply:
This is likely due to the renamer here being severely outdated. Dollars are not valid in PS names. At one point, we supported operators without named aliases, and so it was important to rename `$` to `$dollar`. This obviously has not been the case for _some_ time, but that code was clearly never changed. We can leave this for now.